docs/notebooks/JAX_for_the_impatient.ipynb

```diff
@@ -0,0 +1,1293 @@
+{
```

Comment:
<p>I think someone who knows more than me should review the vjp and jvp sections. (And add a reference to forward-mode and reverse-mode autograd describing in short the computational tradeoff)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google/flax/pull/542/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>

Reply:
<p>Yes, the only thing I can tell is that this is a wide vs tall matrix cost overhead but I'm probably missing some piece of context to make that clearer.</p>